src/Controllers/ElementalAreaController.php

```diff
@@ -216,4 +207,130 @@ public static function removeNamespacesFromFields(array $data, $elementID)
         }
         return $output;
     }
+
+    /**
+     * This method should not be used and will be removed
+     *
+     * Save an inline edit form for a block
+     *
+     * @param HTTPRequest $request
+     * @return HTTPResponse|null JSON encoded string or null if an exception is thrown
+     * @throws HTTPResponse_Exception
+     *
+     * @deprecated 5.3.0 Send a POST request to elementForm/$ItemID instead
+     */
+    public function apiSaveForm(HTTPRequest $request)
+    {
+        Deprecation::notice('5.3.0', 'Send a POST request to elementForm/$ItemID instead');
+        throw new HTTPResponse_Exception('This endpoint should not be used');
```

Comment:
There's no single reason to call this now that we're using FormSchema to save data.  Previously this method handled POSTed form data

If for some reason we wanted to keep this working (which I recommend we do not), then at some point a while back this code did work, though possibly in no longer does

```php
$data = $request->postVars();
$id = $request->param('ID');
$form = $this->getElementForm($id);
if (!$form) {
    $this->jsonError(400);
}
// If the following fails validation, a FormSchema response will be returned to the frontend
// due to a ValidationException being thrown and then caught in core code
$this->save($data, $form);
// Send legacy response
return $this->jsonSuccess(200, [
    'status' => 'success',
    'updated' => $this->elementUpdated,
]);
```

And also we'd need to un-delete any code related to `$elementUpdated` in the PR

Reply:
Throwing an exception here is effectively the same as removing the method. That said, I don't think there's any valid use case for sending manual POST requests for saving blocks. We're probably safe doing this - though it's stretching semver a little.